Neo4j.Driver/Neo4j.Driver/Internal/Protocol/BoltProtocolV3.cs

```diff
@@ -199,8 +201,9 @@ public async Task BeginTransactionAsync(IConnection connection, BeginTransaction
             null,
             null,
             null,
-            Config.Infinite,
-            false);
+            Config.Infinite, // Bolt Protocol 3 uses PULLALL so fetch size is ignored.
```

Comment:
I saw this and had a mild panic then realized why it was infinite so I added this for posterity.